07_deploy_masters.sh

```diff
@@ -73,4 +73,19 @@ for node in $(oc --config ocp/auth/kubeconfig get nodes --no-headers | sed -e 's
   oc label node $node node-role.kubernetes.io/worker=''
 done
 
+#Verify Ingress controller functionality
+echo "Verifying Ingress controller functionality, first we'll check that router pod responsive"
+until curl -o /dev/null -kLs --fail  tst.apps.${CLUSTER_DOMAIN}:1936/healthz; do sleep 20 && echo "Router pod not responding"; done
+
+echo "Create service named tst-svc"
```

Comment:
There is already a lot of valid apps deployed in the cluster - like console - lets check that these are available?

if we want to keep the 'run arbitrary pod' scenario we'd have to take care of retries and error handling, which is terrible in bash

Reply:
@vrutkovs, done. 
Changed the routing test to check console service.